README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to [localhost:8080/]()
+
+Now start changing files in /src and refresh the browser.
```

Comment:
reading this over a second time, we probably want something like "You can now start changing files in the `./src` directory, such that saving your changes and refreshing the browser will show your changes.